### PR TITLE
Add "The Go Programming Language"

### PR DESCRIPTION
**Pull request by adonovan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,27 @@ Intended for seasoned Go programmers who want to put their expertise in Go to us
...
+Alan A. A. Donovan and Brian W. Kernighan show you how to write clear
+and idiomatic Go to solve real-world problems.
+The book does not assume prior knowledge of Go nor experience with
+any specific language, so you'll find it accessible whether you're
+most comfortable with JavaScript, Ruby, Python, Java, or C++.
+
+The book features hundreds of interesting and practical examples of
+idiomatic Go code that cover the whole language, its most important libraries,
+and a wide range of applications.
+Source code is freely available for download from the book's companion web site `gopl.io`,
```
Comment (nathany) [inline README.md]: I would suggest making this a link `[gopl.io](http://www.gopl.io/)`

**Comment by nathany** [comment]
Nice website, and nice new cover. October 30th. Whoo hoo!

**Comment by alandonovan** [comment]
Good idea; done.

**Comment by nathany** [comment]
LGTM

/cc @dariubs

**Comment by adonovan** [comment]
Are you going to merge this branch?  I don't have write access.

**Comment by dgryski** [comment]
@adonovan only @dariubs has a commit bit on this repo, I think.

**Comment by nathany** [comment]
That's correct. I don't have a commit bit.

**Comment by nathany** [comment]
Merged. Thanks @dariubs

**Pull request merged**